src/Symfony/Component/Cache/Traits/Redis6ProxyTrait.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Traits;
+
+if (version_compare(phpversion('redis'), '6.0.2', '>')) {
```

Comment:
I wonder if we should compare with `6.1.0` instead (because of phpredis/phpredis#2530)

Reply:
A first RC for 6.1 has been tagged. I have updated the version tag accordingly here.